maistra/run-ci.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+set -x
+
+ARCH=$(uname -p)
+if [ "${ARCH}" = "ppc64le" ]; then
+  ARCH="ppc"
+fi
+export ARCH
+
+export BUILD_SCM_REVISION="Maistra PR #${PULL_NUMBER:-undefined}"
+export BUILD_SCM_STATUS="SHA=${PULL_PULL_SHA:-undefined}"
+
+# Build
+bazel build //source/exe:envoy-static
+
+echo "Build succeeded. Binary generated:"
+bazel-bin/source/exe/envoy-static --version
+
+# Run tests
+bazel test //test/...
```

Comment:
Could you add --build_tests_only here? Otherwise this will attempt to build everything under /test (e.g. fuzzer tests, which are not based on gunit): which is slower at best, may fail at worst.

Reply:
Done